### PR TITLE
make sure inbound sms page doesn't break if you receive a blank inbou…

### PR DESCRIPTION
**Pull request by leohemsted**
…nd message

### CODE REVIEW

File: app/main/views/conversation.py [hunk 1]
```diff
@@ -59,8 +59,14 @@ def get_user_number(service_id, notification_id):
...
+            dict(type='outbound', **x) for x in
+            notification_api_client.get_notifications_for_service(service_id, to=user_number)['notifications']
+        ] +
+        [
+            dict(type='inbound', **x) for x in
+            service_api_client.get_inbound_sms(service_id, user_number=user_number)
+        ]
     ), key=lambda notification: notification['created_at']):
 
         is_inbound = ('notify_number' in notification)
```
Comment (quis) [inline app/main/views/conversation.py]: This should go now if `notification['type']` means the same thing. Or should be `is_inbound = notification['type'] == 'inbound'` if there’s a good reason to keep it.

**Comment by quis** [comment]
Also, what does the empty message look like on the screen?

**Comment by leohemsted** [comment]
it looks: pretty bad

image


image

**Comment by quis** [comment]
it looks OK

**Pull request merged**